pandas/core/arrays/string_.py

```diff
@@ -325,6 +325,12 @@ def _validate(self):
                 "StringArray requires a sequence of strings or pandas.NA. Got "
                 f"'{self._ndarray.dtype}' dtype instead."
             )
+        # Check to see if need to convert Na values to pd.NA
+        if self._ndarray.ndim > 2:
+            # Ravel if ndims > 2 b/c no cythonized version available
+            lib.convert_nans_to_NA(self._ndarray.ravel("K"))
```

Comment:
isnt the ravel unnecessary bc convert_nans_to_NA supports 2D?

Reply:
This is for > 2D (e.g. 3d stuff). IIUC, pandas doesn't support > 2D but the fancy indexing tests that check for an error in the 3D case, expect that a 3D StringArray creates correctly for some reason only to raise later in the series constructor.

Reply:
can we just always ravel and then reshape?

Reply:
for all ndim

Reply:
> can we just always ravel and then reshape?

risks making copies

Reply:
kk, don't luv the ndim > 3 here at all (i get your reasoning) but as this will be refactored would be good to simplify.